src/main/java/appeng/items/parts/PartType.java

```diff
@@ -57,7 +57,8 @@
 import appeng.parts.networking.PartCableCovered;
 import appeng.parts.networking.PartCableGlass;
 import appeng.parts.networking.PartCableSmart;
-import appeng.parts.networking.PartDenseCable;
+import appeng.parts.networking.PartDenseCableCovered;
+import appeng.parts.networking.PartDenseCableSmart;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'appeng.parts.networking.PartDenseCableCovered' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)